AI/VCAI/Goals/Conquer.cpp

```diff
@@ -58,8 +59,14 @@ Tasks::TaskList Conquer::getTasks() {
 	// lets process heroes according their army strength in descending order
 	std::sort(heroes.begin(), heroes.end(), isLevelHigher);
 
-	addTasks(tasks, sptr(CaptureObjects().ofType(Obj::TOWN)), 1);
-	addTasks(tasks, sptr(CaptureObjects().ofType(Obj::HERO)), 0.95);
+	addTasks(tasks, sptr(Defence()), 1);
```

Comment:
Making Defense part of "Conquer" is questionable. In some scenarios AI might be able to win the game immediately without defending or passing turn.

Reply:
Win goal should take care of it. Priorities are local to current goal.